types/mparticle__web-sdk/index.d.ts

```diff
@@ -362,6 +362,10 @@ interface SelectPlacements {
     (options: RoktSelectPlacementsOptions): Promise<RoktSelection>;
 }
 
+interface Use {
+    (name: string): Promise<unknown>;
+}
+
```

Comment:
I know I had approved this, but I was looking at it again, and I think this should be 
```suggestion
interface Use {
    <T>(name: string): Promise<T>;
}
```

Since the return type is a generic.  We may want to confirm with Matt. As I'm not clear on what extensions currently return?